src/java/picard/analysis/directed/TargetMetricsCollector.java

```diff
@@ -109,16 +116,35 @@
 
     private final long genomeSize;
 
+    private final int minimumMappingQuality;
+
```

Comment:
I try to follow the style in the rest of the file.  In the diff here it doesn't show the extra lines elsewhere.